src/main/java/io/github/thebusybiscuit/exoticgarden/PlantsListener.java

```diff
@@ -54,8 +55,9 @@ public void onGrow(StructureGrowEvent e) {
 
         if (item != null) {
             e.setCancelled(true);
-            if (!e.getLocation().getChunk().isLoaded()) e.getLocation().getWorld().loadChunk(e.getLocation().getChunk());
-
+            if (!PaperLib.isChunkGenerated(e.getLocation())) {
```

Comment:
Also this PaperLib call should probably be conditional since Paper may not be used actually, so adding this very hard constraint to all servers is not a good idea.